pkg/rtc/participant.go

```diff
@@ -73,6 +73,7 @@ type ParticipantParams struct {
 	Migration                    bool
 	AdaptiveStream               bool
 	AllowTCPFallback             bool
+	TCPFallbackRttThreshold      int
```

Comment:
nit: even tho it reads weird, following Go naming of using caps for abbreviations for consistency
```suggestion
	TCPFallbackRTTThreshold      int
```